sgraph_show_with_vertex_coloring.py

```diff
@@ -0,0 +1,37 @@
+# Show an SGraph with custom vertex coloring
+import graphlab as gl
+import networkx as nx # Requires 'pip install networkx==1.9.1'
+
+# We will use the "Tableau 20" color scheme as RGB triples scaled to 0-1
+# See http://public.tableausoftware.com/profile/chris.gerrard#!/vizhome/TableauColors/ColorPaletteswithRGBValues
+tableau20 = [[0.12156862745098039, 0.4666666666666667, 0.7058823529411765],
```

Comment:
Might it be possible to pull this in automatically rather than enumerating it in the how-to?